test/conformance/runtime/filesystem_test.go

```diff
@@ -55,8 +55,12 @@ func testFiles(t *testing.T, clients *test.Clients, paths map[string]types.FileI
 			return fmt.Errorf("%s.Error = %s, want: %s", path, riFile.Error, file.Error)
 		}
 
-		if file.IsDir != nil && *file.IsDir != *riFile.IsDir {
-			return fmt.Errorf("%s.IsDir = %t, want: %t", path, *riFile.IsDir, *file.IsDir)
+		if file.IsDir != nil {
+			if riFile.IsDir == nil {
+				return fmt.Errorf("%s.IsDir = %t, want: %t", path, nil, *file.IsDir)
```

Comment:
just say `"%s.IsDir was nil"`

Reply:
I moved nil into the format string, but I would like to leave the desired value in there.